packages/esm-ward-app/src/hooks/useEmrConfiguration.ts

```diff
@@ -0,0 +1,30 @@
+import { type FetchResponse, openmrsFetch, type OpenmrsResource, restBaseUrl } from '@openmrs/esm-framework';
+import { useMemo } from 'react';
+import useSWRImmutable from 'swr/immutable';
+
+interface EmrApiConfigurationResponse {
+  admissionEncounterType: OpenmrsResource;
+  clinicianEncounterRole: OpenmrsResource;
+  consultFreeTextCommentsConcept: OpenmrsResource;
+  visitNoteEncounterType: OpenmrsResource;
+  // There are many more keys to this object, but we only need these for now
+  // Add more keys as needed
+}
+
+export default function useEmrConfiguration() {
+  const swrData = useSWRImmutable<FetchResponse<EmrApiConfigurationResponse>>(
+    `${restBaseUrl}/emrapi/onfiguration`,
```

Comment:
```suggestion
    `${restBaseUrl}/emrapi/configuration`,
```

typo?

Reply:
Actually, I was testing failure case and forgot to fix it, I have fixed it now.
Thanks for the catch!